### ISSUE TITLE
footer in github doesn't coincide with footer in Disco2EB

### ISSUE THREAD

**Issue by bernardo-gomez** [emory-libraries/discovere-restyle#4]
kristian,
is github code going to be discoverE production? 

footer in in http://emory-primosb-alma.hosted.exlibrisgroup.com/Disco2EB:default_scope:01EMORY_ALMA21200366310002486
seems to reflect expected code.
for instance, sendtophone link is 
sendtophone_bg="https://kleene.library.emory.edu/cgi-bin/sendtophone?doc_id="+merged_record_id;  (OK)
BUT same line in github footer  is 
sendtophone_bg="https://mail.library.emory.edu/cgi-bin/sendtophone?doc_id="+merged_record_id;
which comes from an outdated footer file. ("mail.library.emory.edu")
bottom line: footer in github should reflect current vid=discovere production site,  if github is going to control production version.

**Comment by kristianserrano** [comment]
@bernardo-gomez would you like to make the edit in a branch and submit the PR?

**Comment by kristianserrano** [comment]
Fixed.
